simpleviewinc/keg-cli
# Add svkeg repos

## Context

* Github NPM packages are a pain to work with
* They always require a git key
* They require a .npmrc to overwrite the default settings

## Goal

* Use the `@svkeg` organization
* Updated the dependencies to use the `@svkeg` org

## Updates

* `.npmrc`
  * Removed
* `package.json`
  * Updated to use the `@svkeg` organization
* `...all other files`
  * Updated to use `@svkeg` for all publish repos
  * See https://www.npmjs.com/search?q=%40svkeg

## Testing

* Pull down branch
  * `keg pr 25`
* Install all deps
  * `yarn install`
  * Ensure all repos install properly
    * There should be no errors
* Run tests
  * `yarn test`
* Ensure all tests pass
* Use the keg-cli to start some containers
  * Ensure tasks run as expected
